lighthouse/classes/services/mongo.py

```diff
@@ -0,0 +1,28 @@
+from flask import current_app as app
```

Comment:
I think the convention is also to name the file after the class it contains, separated with underscores.